lib/ts/recipe/accountlinking/recipe.ts

```diff
@@ -670,7 +670,6 @@ export default class Recipe extends RecipeModule {
                                 // See comment about tenantId in the createEmailVerificationToken params
                                 tenantId: input.user.tenantIds[0],
                                 token: resp.token,
```

Comment:
the implementation related to [this comment](https://github.com/supertokens/supertokens-auth-react/pull/780#discussion_r1515948343) is not yet done right? Or am i missing something?

Reply:
I've just added it, I was trying to resolve all other comments first.